rclcpp/include/rclcpp/exceptions.hpp

```diff
@@ -226,6 +226,21 @@ class InvalidParameterValueException : public std::runtime_error
   using std::runtime_error::runtime_error;
 };
 
+/// Thrown if requested parameter type is invalid.
```

Comment:
You might mention this is essentially just `rclcpp::ParameterTypeException` except it has the parameter name associated with it as well.

Reply:
Done 23fbb04